ui/src/app/shared/pod-name.ts

```diff
@@ -55,7 +55,7 @@ export const getTemplateNameFromNode = (node: NodeStatus): string => {
     }
 
     // fall back to v1 pod names if no templateName or templateRef defined
-    if (!node.templateRef) {
+    if (node?.templateRef === undefined || node?.templateRef.template === '') {
```

Comment:
the `node?.` part is not necessary; the `node.templateName` line above this would error out before reaching this line if it didn't exist.

#7605 did fix this, it just never got backported to 3.2.x. So #9167 was a pure duplicate.

This whole function can actually be simplified into a one-liner, which I'm working on.

Reply:
Simplified in #12964